app/tests/conftest.py

```diff
@@ -21,24 +22,30 @@ def content_model() -> PromptForClassification:
 
 
 @pytest.fixture(scope="session")
-def problem_df(path: str = "app/static/user_answer.csv") -> pd.DataFrame:
+def user_answer_df(
+    path: str = "app/static/changed_user_answer.csv",
+) -> pd.DataFrame:
     return pd.read_csv(path)
 
 
-@pytest.fixture(scope="session")
-def problem_dict(keyword_model: SentenceTransformer, problem_df: pd.DataFrame) -> dict:
+@pytest.fixture(scope="function")
+def problem_dict(keyword_model: SentenceTransformer, user_answer_df) -> dict:
     problem_dict = {}
-    keyword_id = 0
-    # criterion parsing
-    for _, data in problem_df.iterrows():
+
+    for _, data in user_answer_df.iterrows():
         problem_id = data["problem_id"]
+        keyword_id = problem_id * 10
         if problem_id not in problem_dict:
             keyword_standards = []
-
-            for criterion in eval(data["keyword_criterion"]):
-                content, _ = map(str.strip, criterion.split("-"))
-                keyword_standards.append(KeywordStandard(id=keyword_id, content=content))
+            for content in eval(data["keyword_criterion"]):
                 keyword_id += 1
+                if content is nan:
+                    content = "NULL"
+                if "," in content:
+                    for split_content in content.split(","):
+                        keyword_standards.append(KeywordStandard(id=keyword_id, content=split_content))
+                else:
+                    keyword_standards.append(KeywordStandard(id=keyword_id, content=content))
```

Comment:
얘는 굳이 "," 체크를 안해도 될 것 같아요!

```suggestion
                for split_content in content.split(","):
                    keyword_standards.append(KeywordStandard(id=keyword_id, content=split_content))
```

Reply:
그러네요!! 적용할게요 👍